src/System.Net.Sockets.Legacy/tests/FunctionalTests/SelectAndPollTests.cs

```diff
@@ -15,19 +16,19 @@ namespace System.Net.Sockets.Tests
     public class SelectAndPollTests
     {
         const int SelectTimeout = 100;
-        const int SelectSuccessTimeout = 5000;
+        const int SelectSuccessTimeoutMicroseconds = 5000000;
```

Comment:
nit: add comment at end of line something like
```c#
// 5 seconds
```

All those zeroes are hard to visually parse.